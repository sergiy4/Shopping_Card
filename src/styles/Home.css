

@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@300;500;900&display=swap');

    
.Start_screen{
    
    /* background-color: rgb(37, 255, 164); */
    background-color: rgb(88, 196, 136);
    height: 85%;
    display: flex;
    
    justify-content: center;
    align-items: center;
    position: relative;
    font-family: 'Raleway', sans-serif;
   
}

.Main_text{
    position: relative;
}
.Home_text_block{
    height: 80%;
    width: 90%;

    display: flex;
    justify-content:space-around;
    align-items: center;
    position: relative;

    font-size: 30px;
    background-color: rgb(88, 196, 136);
    color:rgb(54, 163, 105);

    transform-style: preserve-3d;
  
}
.Around_shop_button{
    width: 202px;
    padding-bottom: 18px;
    margin-top: 40px;
    
   padding-right: 10px;
}
.Shop_Btn{
  
    height: 90px;
    width: 200px;
    transform-style: preserve-3d;
    z-index: 999;
   cursor: pointer;
   font-size: 20px;
   letter-spacing: 6px;
   background-color: #ddd;
   position: relative;
  
}

.block{
  
    z-index: 1;
  
}
@media screen and (max-width: 1000px){
    .Home_text_block{
      
        font-size: 15px;
      
       
    }
    .Around_shop_button{
        width: 110px;
       
        padding-bottom: 18px;
        margin-top: 40px;
        
       padding-right: 10px;
    }
    .Shop_Btn{
      
        height: 70px;
        width: 100px;
        transform-style: preserve-3d;
        z-index: 999;
       cursor: pointer;
       font-size: 15px;
       letter-spacing: 6px;
       background-color: #ddd;
       position: relative;
      
    }
}