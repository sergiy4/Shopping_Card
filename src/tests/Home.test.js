import {render, screen,cleanup} from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import Home from '../components/HomePage'
import "@testing-library/jest-dom"
afterEach(()=>{
    cleanup()
})

describe('Testing HomePages',()=>{
    test("renders",()=>{
        const {asFragment} = render(<Home/>)

        expect(asFragment()).toMatchSnapshot()
    })

    test("Renders component h1",()=>{
       render(<Home/>);

        const h1 = screen.getByText('Good')

        screen.debug();
        expect(h1).toBeInTheDocument()
        
    })


    test("Renders img flowers",()=>{
        render(<Home/>);

        const BtnShop = screen.getByText('Shop');


        expect(BtnShop).toBeInTheDocument()
    })
})

// I saw this flower and thought of you because it's pretty and... Well, I don't really like it, but I thought you might like it, 'cause you're pretty.