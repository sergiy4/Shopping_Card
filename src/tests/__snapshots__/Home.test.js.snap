// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HomePages renders 1`] = `
<DocumentFragment>
  <main
    class="Start_screen"
  >
    <div
      class="Home_text_block"
    >
      <div
        class="block"
      >
        <div>
          <h1
            class="Main_text"
            style="text-shadow: 0px 0px 0px  rgba(0, 0, 0, 1); top: 0%; left: 0%; transform: none;"
          >
            <span
              class="Main_span first_spain"
            >
              Good
            </span>
             
            <br />
             Plant - good 
            <br />
            <span
              class="Main_span"
            >
              Mood
            </span>
          </h1>
          <button
            class="Shop_Btn"
            style="box-shadow: 0px 0px 0px 0px rgba(0, 0, 0, 1); top: 0%; left: 0%;"
          >
            Shop
          </button>
        </div>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
